tests/virtualization/xen/storage.pm

```diff
@@ -23,8 +23,11 @@ sub run_test {
     my ($self) = @_;
     my $hypervisor = get_var('HYPERVISOR') // '127.0.0.1';
 
-    record_info "Default pool";
-    assert_script_run "virsh pool-list --all | grep default | grep active";
+    #VIRT_AUTOTEST do not have default pool
+    if (!get_var('VIRT_AUTOTEST')) {
+        record_info "Default pool";
+        assert_script_run "virsh pool-list --all | grep default | grep active";
+    }
```

Comment:
You can even remove this. I discovered that the `default` pool is created by `virt-manager` but it's not very important.

Reply:
Done